### DISCUSSION TITLE
Strange paths in terminal after uninstalling homebrew

### DISCUSSION THREAD

**Discussion by jaspermowatt** [Homebrew/discussions#1016] [Everyday usage]
Hi,

I installed homebrew a couple months ago but then I decided to uninstall it. It seemed to uninstall fine but now when I open terminal on my mac I get the following:
**_Last login: Mon Mar  8 20:29:03 on ttys000
/Users/jasper/.zshrc:export:2: not valid in this context: point:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/VMware Fusion.app/Contents/Public:/opt/X11/bin:/Applications/Postgres.app/Contents/Versions/latest/bin
jasper@Jaspers-MBP ~ %_** 

I was hoping someone could help me understand what this is saying and how I can change it back to normal.
To note, the two programs VM Fusion and Postgres have both been removed from my computer.

I am not a computer whizz so I am pretty stuck here.

If I use sudo, curl or pip in the terminal, it says none of these commands can be found which also confuses me.

Any help is really appreciated. Thanks

**Comment by SMillerDev** [comment]
When you installed brew you needed to run 2 commands at the end of the install. They wrote something to the `.zshrc` file in your user directory. You can run `open ~/.zshrc` to edit that file and remove the reference to brew. That should fix it.

**Reply by jaspermowatt** [reply]
Thanks, but when I run that, it doesn't recognise open:
Last login: Tue Mar  9 08:17:33 on ttys000
/Users/jaspermowatt/.zshrc:export:2: not valid in this context: point:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/VMware Fusion.app/Contents/Public:/opt/X11/bin:/Applications/Postgres.app/Contents/Versions/latest/bin
jaspermowatt@Jaspers-MBP ~ % open ~/.zshrc
zsh: command not found: open

**Reply by SMillerDev** [reply]
Okay, how about `/usr/bin/open ~/.zshrc` or `/usr/bin/nano ~/.zshrc`

**Reply by jaspermowatt** [reply]
/usr/bin/open ~/zzshrc results in:

```
export PATH=copied path from the above point:$PATH

# >>> conda initialize >>>
# !! Contents within this block are managed by 'conda init' !!
__conda_setup="$('/Users/jaspermowatt/opt/anaconda3/bin/conda' 'shell.zsh' 'hook' 2> /dev/null)"
if [ $? -eq 0 ]; then
    eval "$__conda_setup"
else
    if [ -f "/Users/jaspermowatt/opt/anaconda3/etc/profile.d/conda.sh" ]; then
        . "/Users/jaspermowatt/opt/anaconda3/etc/profile.d/conda.sh"
    else
        export PATH="/Users/jaspermowatt/opt/anaconda3/bin:$PATH"
    fi
fi
unset __conda_setup
# <<< conda initialize <<<
```

I am unsure what this is showing though

**Reply by gromgit** [reply]
>```
>export PATH=copied path from the above point:$PATH
>```

Please don't edit command output like this. Non-techies may think a character or two irrelevant and can therefore be left out, but they often turn out to make all the difference in the world.

Please copy and paste the above line from your ~/.zshrc, _exactly_.
